vite.config.ts

```diff
@@ -1,6 +1,17 @@
+import 'dotenv/config';
 import { sveltekit } from '@sveltejs/kit/vite';
 import { defineConfig } from 'vite';
 
-export default defineConfig({
-	plugins: [sveltekit()]
-});
+const developmentAllowedEnvironmentVariables = [
+  'DEFENDER_POOL_ID',
+  'DEFENDER_POOL_CLIENT_ID',
+  'DEFENDER_API_URL',
+];
+
+export default defineConfig(({ mode }) => ({
+	plugins: [sveltekit()],
+	define: mode === "development" ? developmentAllowedEnvironmentVariables.reduce((definedEnvironmentVariables, allowedEnvironmentVariable) => ({
+		...definedEnvironmentVariables, 
+		[`process.env.${allowedEnvironmentVariable}`]:JSON.stringify( process.env[allowedEnvironmentVariable]) 
+	}), {}) : undefined,
+}));
```

Comment:
- When there is no ambiguity, shorter variable names can make it easier to visualize what is used where.  It's hard to visually differentiate between long variable names.
- When the intent is to iterate through every element to do something, `.forEach` makes this more obvious.
```suggestion
const developmentEnvironmentVariables = [
  'DEFENDER_POOL_ID',
  'DEFENDER_POOL_CLIENT_ID',
  'DEFENDER_API_URL',
];

export default defineConfig(({ mode }) => {
  const defineVars: Record<string, string> = {};

  if (mode === "development") {
    developmentEnvironmentVariables.forEach(v => {
      defineVars[`process.env.${v}`] = JSON.stringify(process.env[v]);
    });
  }

  return {
    plugins: [sveltekit()],
    define: mode === "development" ? defineVars : undefined,
  };
});
```